### ISSUE TITLE
FT: Statistics

### ISSUE THREAD

**Issue by gerroon** [zadam/trilium#785]
Hi

I think that it would be very useful to know more about the data stored. For instance how big is the notebooks, number of files, largest stored file, number of folders,  number of notes, number of journals etc etc. Note apps generally provide such statistics. Obviously this is not a necessity but it helps with getting a grasp of what is going on with the data.

**Comment by zadam** [comment]
Yes, I plan such functionality, but not as part of the project itself but in the form of script (package), also included in the demo document. It would be nice showcase of scripting ...
